src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/CustomFx2NLParams.cs

```diff
@@ -20,7 +20,12 @@ public class CustomFx2NLParams : BaseNLParams, IHasTextDocument
         /// <summary>
         /// Existing Power Fx Expression.
         /// </summary>
-        public string Expression { get; set; }
+        public string Expression { get; set; }
+
+        /// <summary>
+        /// Optional range of the expression.
```

Comment:
>range of the expression. [](http://example.com/codeflow?start=21&length=24)

doc comments can refer to other fields :) Something like <see>

Comment: 
range within <see="Expression">  to explain.
If missing, then explain the whole expression.